File: official/docs/python/insurances/create_insurance.py
import easypost


easypost.api_key = "EASYPOST_API_KEY"

insurance = easypost.Insurance.create(
    to_address={"id": "adr_..."},
    from_address={"id": "adr_..."},
    tracking_code="9400110898825022579493",
    carrier="USPS",
    amount="100.00",
    reference="insuranceRef1",
)

print(insurance)

File: official/docs/ruby/insurances/create_insurance.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

insurance = EasyPost::Insurance.create(
  {
    to_address: {
      id: 'adr_...',
    },
    from_address: {
      id: 'adr_...',
    },
    tracking_code: '9400110898825022579493',
    carrier: 'USPS',
    reference: 'insuranceRef1',
    amount: '100.00',
  },
)

puts insurance
